mirrord/cli/src/config.rs

```diff
@@ -182,6 +183,122 @@ pub(super) struct ExecArgs {
     pub context: Option<String>,
 }
 
+impl ExecParams {
+    pub fn to_env(&self) -> Result<HashMap<String, OsString>, CliError> {
```

Comment:
```suggestion
    pub fn into_env_vars(self) -> Result<HashMap<String, OsString>, CliError> {
```

Do we ever want to keep `ExecParams` alive after using this?

Reply:
Also add some docs, something like

```rust
/// Creates a `HashMap` of environment variables based on the command params passed by the user.
```

Reply:
Into is a bit problematic because we don't always can partially destruct the parent of `ExecParams` (`ExecArgs` & `ContainerArgs`)